### ISSUE TITLE
Add integration test to test data-integration and compliance to the dataprofile

### ISSUE THREAD

**Issue by SvenLieber** [kbrbe/beltrans-data-integration#137]
**TL;DR several software integration tests cover the data integration and separately the corpus SPARQL query according to our dataprofile, yet another test is needed to test the combination of both.**

For the dynamic generation of SPARQL queries for the data integration, we also developed integration tests. These tests contain test data and all tests succeeded for both implementations, integration based on serialized SPARQL queries and dynamically generated SPARQL queries based on a config.

However, by using the Jupyter notebook statistics of the corpus as well as the corpus Excel sheet itself we still found issues when comparing it to the older corpus version. These issues were fixed as well.

This means that our tests do not cover all important aspects yet. We have to add another test, testing not only a proper integration of identifiers, but also the resulting corpus according to the dataprofile. E.g. to verify that besides the integration based on identifiers, correct RDF properties are used and all relevant values from the source were added as well. 

This can be done by e.g. reusing the [dataprofile ](https://github.com/kbrbe/beltrans-data-integration/blob/01ef85e572c4ab01287862de6cf89fd2845d66a8/data-integration/test/testDataprofileQueryThreeSources.py) test case, i.e. test another implementation which instead of testing the dataprofile query based on already prepared integrated and source data will test the dataprofile query based on only source data and our integration scripts.
